mlflow/R/mlflow/tests/testthat/test-model.R

```diff
@@ -52,27 +52,6 @@ test_that("mlflow can save model function", {
   )
 })
 
-test_that("mlflow can save/log model with dependencies", {
```

Comment:
Makes sense to remove this since crate no longer takes conda dependencies - ideally we would still test logging a model with dependencies somewhere (e.g. in the keras model tests), but we can do that in a follow-up